pynamodb/attributes.py

```diff
@@ -473,6 +475,13 @@ def deserialize(self, value):
         return None
 
 
+class MapAttributeMeta(AttributeContainerMeta):
+    """
+    This is only here for backwards compatibility: i.e. so type(MapAttribute) == MapAttributeMeta
+    """
```

Comment:
don't you need to add a `pass` in here? I am surprised this is syntactically correct in python 2.7.